package exceptiontest;

public class ZeroValueException extends Throwable{
	
	
	public ZeroValueException() {
		System.out.println("Can't divided by zeo");
	}

	
}
